common-libs/common/src/services/KeysService.ts

```diff
@@ -9,6 +8,7 @@ import { convertDecryptedSeedBufferToString, parseDecryptedSeed } from '../share
 import DigestService from './DigestService'
 import DidDocumentService from './DidDocumentService'
 
+const createHash = require('create-hash/browser')
```

Comment:
why we use require? Is it not outdated? There are not type support with require,  as I know , that could lead to mistakes and typos.
Also, what happen if this code will be used on backend side?

Reply:
Also I don't see any reason for `KeysService` to be affected by this PR.

Reply:
we have a discussion with @DenisPopov15 . it is a bug in sdk so all functions that use createhash will work only in nidejs.

Reply:
so what we should do ? just ignore issue ?

Reply:
And after this fix they are only going to work in browser?
We should have removed this change from this MR and create a separate ticket and fix it _properly_.